src/lib/containers/QueryWrapper.tsx

```diff
@@ -385,7 +398,14 @@ export default class QueryWrapper extends React.Component<
             data?.selectColumns
               ?.slice(0, this.props.visibleColumnCount ?? Infinity)
               .map(el => el.name) ?? [],
+          isFacetsAvailable: isFaceted,
```

Comment:
On initialization, respect the default showFacetFilter and showFacetVisualization value. But if configured to show, then only show if facets are actually available.